tests/selectv_timepart.test/lrl.options

```diff
@@ -1,2 +0,0 @@
-table t1 t1.csc2
```

Comment:
t1 is used in the test and has a valid schema. It looks like the test is failing now because there's no table t1. Did you mean to remove it?